lib/rules/no-duplicate-selectors/__tests__/index.js

```diff
@@ -73,6 +73,14 @@ testRule(rule, {
       line: 1,
       column: 1
     },
+    {
+      code: ".a, .b, .a, .c, .d, .a {}",
```

Comment:
Could you keep only one `.a`? :)

Reply:
It was made in favor of https://github.com/stylelint/stylelint/issues/4172 implementation (if it will be). For now I'll fix it.

Reply:
done